host.go

```diff
@@ -19,6 +19,8 @@ type Client interface {
 	Logger() Logger
 }
 
+// gpbftRunner is responsible for running gpbft.Participant, taking in all concurrent events and
+// passing them to gpbft in a single thread.
 type gpbftRunner struct {
```

Comment:
Would be nice to reduce concept here: gpbftHost/gpbftRunner/Module.

Reply:
Module got renamed F3, host and runner are the same thing just with different APIs exposed

Reply:
> Module got renamed F3

Has this change been pushed in this PR?

Reply:
Uhhh, I accidentally un-did it.